wasm/test_page/bergamot.html

```diff
@@ -206,10 +206,10 @@
     const from = lang.substring(0, 2);
     const to = lang.substring(2, 4);
     let start = Date.now();
-    await constructTranslationModel(from, to);
-    log(`translation model ${from}${to} construction took ${(Date.now() - start) / 1000} secs`);
+    await constructTranslationService(from, to);
+    log(`translation service ${from}${to} construction took ${(Date.now() - start) / 1000} secs`);
     document.querySelector("#load").disabled = false;
-    //log('Model Alignment:', translationModel.isAlignmentSupported());
+    //log('Model Alignment:', translationService.isAlignmentSupported());
```

Comment:
The commenting implies `translationService.isAlignmentSupported()` is not covered by the test-page? 

Do we have any use for this information? Can we agree here to get rid of `isAlignmentSupported()` making it always true, removing the commented line, given the translation model is Marian, and from the looks of it I'm always setting alignment on since from the looks of it tag matching is a "requirement" to properly match links and from the methods chosen, internally we are always supporting (word) alignments. (cc @qianqianzhu, @kpu).

Reply:
It is not used anywhere at the moment. I have no issue in removing `isAlignmentSupported()` if the trained marian models can always return alignment information. The extension will use the returned alignments soon to re-insert tags.

However, this refactoring PR shouldn't be blocked on this. Removing this API can be done as a separate PR.